src/shared/markdown.ts

```diff
@@ -38,6 +40,10 @@ export let customEmojisLookup: Map<string, CustomEmojiView> = new Map<
   CustomEmojiView
 >();
 
+if (isBrowser()) {
+  Tribute = await import("tributejs");
+}
+
```

Comment:
We might just need to add an exception for the require rule, because I don't see any other way around it. Tribute should be smart enough to null check the window object for SSR apps, but it doesn't.